pymc/distributions/continuous.py

```diff
@@ -3121,7 +3121,11 @@ def triangular_default_transform(op, rv):
 
 class Gumbel(Continuous):
     r"""
-    Univariate Gumbel log-likelihood.
+    Univariate Gumbel right Skew log-likelihood.
```

Comment:
```suggestion
    Univariate right skewed Gumbel log-likelihood.
```
Or do we prefer "right-skewed" (hyphenated)?

Reply:
I like your "right-skewed" (hyphenated) suggestion!